component/src/main/java/org/wso2/extension/siddhi/execution/kalmanfilter/KalmanFilter.java

```diff
@@ -162,34 +164,8 @@
     private long prevTimestamp;
```

Comment:
All these class variables should be removed. You are adding these values to the state object, isn't it? Then you have to access the state object to access these values. Please correct it in all places.

Reply:
fixed.